### ISSUE TITLE
The BatchNorm2d running_mean and running_var could not use when i shift the parameter from mxnet model

### ISSUE THREAD

**Issue by zhujun5164** [pytorch/pytorch#39721]
## ❓ Questions and Help

i have written a model by pytorch, and transform the parameter from mxnet。 But the results of two model are different. I have print all inside results and find the batchnorm2d layer could use the running_mean and running_var parameter to compute the input.

the code of running_mean and running_var are :
`sym, arg_params, aux_params = mx.model.load_checkpoint(mx_path, epoch)`
`checkpoint = init_model.state_dict()`
`mx_data = aux_params['batchnorm_monving_mean'].asnumpy()`
`checkpoint[bn.running_mean].copy_(torch.FloatTensor(mx_data))`
`init_model.load_state_dict(checkpoint)`

And i have create a BatchNorm2d layer and copy the parameter from my pytorch parameter that i have transformed the mxnet parameter, the result is true.

**Comment by albanD** [comment]
Hi,

We use github issues only for bugs or feature requests.
Please use the forum to ask questions: https://discuss.pytorch.org/

When asking the question there, please provide a full code sample showing your issue. Thanks !
